tox.ini

```diff
@@ -24,10 +26,23 @@ envlist =
     linkcheck
 
 [testenv:test]
-description = run unit tests
+description = run base set of unit tests with no extra functionality
+deps =
+    pytest>=7
+commands =
+    python --version
+    pytest -v -rs {posargs}
+
+[testenv:test-encryption]
+description = run base unit tests with encryption enabled
+deps =
+    {[testenv:test]deps}
+    pymongocrypt>=1.6.0,<2.0.0
+    pymongo-auth-aws<2.0.0
```

Comment:
is it possible to refer to extras here so we don't need to duplicate the dependencies? like `.[encryption]`?

Reply:
Yeah that should work

Reply:
Fixed, thanks!